network/local.go

```diff
@@ -7,17 +7,14 @@ import (
 	"time"
 )
 
-// NewLocalRouter returns a fresh router which uses only local queues. It uses
-// the default local manager.
-// If you need multiple independent local-queues, use NewLocalRouterWithManager.
-// In case of an error it is returned together with a nil-Router.
+// NewLocalRouter returns a fresh router which uses local connections. It takes
```

Comment:
Everything comments ? It's possible, I maybe messed up because for the `refactor_mocking_local` branch , I mostly worked on `refactor_mocking` (so I could build & test) and then merg after into `refactor_mocking_local` . Since your comments have been merged directly on `refactor_mocking_local` maybe I lost a version ?
I'm happy to arrive at this point of the series of PR where I can build directly on it, it was really a nightmare .. ><